ipn/ipnlocal/serve.go

```diff
@@ -111,6 +113,17 @@ func (s *serveListener) Run() {
 	}
 }
 
+func (s *serveListener) shouldWarnAboutListenError(err error) bool {
+	if !s.b.e.GetLinkMonitor().InterfaceState().HasIP(s.ap.Addr()) {
+		// Machine doesn't have IPv6 enabled (or the IP is still being
```

Comment:
Although the comment says IPv6, this code doesn't do any special handling for that.

Reply:
Yeah, fair. It also would end up handling the userspace-networking path for IPv4, but we stopped that earlier (no need to even get this far if we know we're in userspace mode)